pkg/command/cert.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package command
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/vmware-tanzu/tanzu-plugin-runtime/component"
+	configlib "github.com/vmware-tanzu/tanzu-plugin-runtime/config"
+	configtypes "github.com/vmware-tanzu/tanzu-plugin-runtime/config/types"
+	"github.com/vmware-tanzu/tanzu-plugin-runtime/log"
+
+	"github.com/vmware-tanzu/tanzu-cli/pkg/cli"
+)
+
+var (
+	hostName, caCertPath     string
+	skipCertVerify, insecure string
+)
+
+func newCertCmd() *cobra.Command {
+	var certCmd = &cobra.Command{
+		Use:   "cert",
+		Short: "Manage cert configuration of hosts",
+		Long:  "Manage cert configuration of hosts",
+	}
+	certCmd.SetUsageFunc(cli.SubCmdUsageFunc)
+
+	listCertCmd := newListCertCmd()
+	addCertCmd := newAddCertCmd()
+	updateCertCmd := newUpdateCertCmd()
+	deleteCertCmd := newDeleteCertCmd()
+
+	addCertCmd.Flags().StringVarP(&hostName, "hostname", "n", "", "name of the host")
+	addCertCmd.Flags().StringVarP(&caCertPath, "ca-certificate", "", "", "path to the public certificate")
+	addCertCmd.Flags().StringVarP(&skipCertVerify, "skip-cert-verify", "", "", "skip server's TLS certificate verification (true|false)")
```

Comment:
Why not make this a boolean flag?  Then the user could do `tanzu config cert add --name test.vmware.com  --skip-cert-verify`

We can convert this to a string when setting the config file

Reply:
We can use the boolean flag for `--skip-cert-verify` (Infact i initially implemented that way), but for update command we should be able to differentiate whether user provided flag or not, because defaulting to (say `false`) in update command would have bad sideeffects. So, I used empty string to differentiate the user's no input as apposed to boolean value. Hence I used the same  types for `add` command as well.

Reply:
Oh, that makes sense!  Thanks for the explanation.